src/pages/SearchPage/index.tsx

```diff
@@ -169,7 +169,6 @@ function SearchPage({betas, reports, isSearchingForReports, navigation}: SearchP
                             onChangeText={setSearchValue}
                             headerMessage={headerMessage}
                             onLayout={setPerformanceTimersEnd}
-                            autoFocus
```

Comment:
@shubham1206agra I agree that `autoFocus` is redundant here. But is there any discussion about this change?

Reply:
I confirmed `autoFocus` and `shouldDelayFocus` are redundant. Because this change hasn't been discussed, please check why don't we use this param anymore. Is there any new pattern to handle `autoFocus` and `shouldDelayFocus`?